app/config/config.yml

```diff
@@ -114,12 +114,12 @@ swiftmailer:
     spool:     { type: memory }
 
 # Guzzle configuration
-csa_guzzle:
-    profiler:
-        enabled: "%kernel.debug%"
+love_oss_guzzle5:
```

Comment:
Seen with @eternoendless and @PierreRambaud : this is a BC break as it will fail for people using this configuration key.

Example in this PR: https://github.com/PrestaShop/PrestaShop/pull/13761/files#diff-95a1b476935b61587950fe999d326bbaR165
If someone was using `$clientConfig = $config['csa_guzzle']['clients']['addons_api']['config'];` it will fail after this PR

Reply:
@mickaelandrieu Unfortunately because of this issue, we'll have to re-create a fork of CsaGuzzleBundle but this time we'll try to make it as close as possible to the original project: no namespace change, no config key change. The aim is really to keep the code updated and backward compatible

In order to do this, a PrestaShop fork is on the way : https://github.com/PrestaShop/CsaGuzzleBundle

Reply:
I'm not sure we have the right to do that (use the author's name) and this will confuse people who will try go get support to the original github repository

Reply:
🆙 @ttoine do you have any advice on this authoring matter ? 🤔

Reply:
please see my review.